### DISCUSSION TITLE
How to convert a unordered_map to json?

### DISCUSSION THREAD

**Discussion by Miko321** [nlohmann/json#3398] [Q&A]
i have a `unordered_map` defined like:
`std::unordered_map <wchar_t*, std::unordered_map<HWND, LPTSTR>> controlList;`

This is an example of how the object is filled:
![image](https://user-images.githubusercontent.com/98557041/159170343-f69731e3-a452-4ee5-a395-a2d51502bcae.png)

How I could convert this object to JSON?

**Comment by Miko321** [comment]
it's not possible to convert an `unodered_map`?

**Comment by Miko321** [comment]
Could i even get an answer?

**Reply by nlohmann** [reply]
See https://json.nlohmann.me/features/arbitrary_types/

**Reply by Miko321** [reply]
> See https://json.nlohmann.me/features/arbitrary_types/

There's nothing about map in this link, could you share a working example please?
